src/css/buttons.css

```diff
@@ -0,0 +1,36 @@
+.radio__container {
+    @apply block h-6 w-6 relative cursor-pointer;
+    border-radius: 50%;
+}
+
+/* Hide the browser's default radio button */
+.radio__input {
+    @apply opacity-0;
+}
+
+/* Create a custom radio button */
+.radio__checkmark {
+    @apply flex justify-center items-center absolute top-0 left-0 h-6 w-6 bg-white border border-solid border-primary;
+    border-radius: 50%;
+}
+
+/* On mouse-over, add a grey background color */
+.radio__container:hover .radio__input~.radio__checkmark {
```

Comment:
you can use hover-group with tailwind classes instead of using normal css selector

Reply:
Yes, I wasn't aware it was there. Made my lie easier.